web/app/charts/MapEditor.js

```diff
@@ -0,0 +1,457 @@
+if(!bluewave) var bluewave={};
+if(!bluewave.charts) bluewave.charts={};
+
+//******************************************************************************
+//**  MapEditor
+//******************************************************************************
+/**
+ *   Panel used to create Map charts
+ *
+ ******************************************************************************/
+
+ bluewave.charts.MapEditor = function(parent, config) {
+
+    var me = this;
+    var defaultConfig = {
+        style: {
+        }
+    };
+    var margin = {
+        top: 15,
+        right: 5,
+        bottom: 65,
+        left: 82
+    };
+    var chartConfig = {};
+    var svg;
+    var panel;
+    var previewArea;
+    var optionsDiv;
+    var mapArea;
+    var currentNode;
+    var mapLayer;
+    var inputData = [];
+    var mapInputs = {
+        projection:null,
+        mapType:null,
+        lat:null,
+        long:null,
+        mapValue:null
+    };
+    var mapProjection;
+    var projectionOptions;
+
+
+  //**************************************************************************
+  //** Constructor
+  //**************************************************************************
+    var init = function(){
+        projectionOptions = [
```

Comment:
What happened to the other projections? Like geoMercator()?

Reply:
All of these projections were removed after a conversation with you. We use the map level to decide the projection now. The end user doesn't even know about the projections.